src/ui/components/CheckboxBoolean/CheckboxBoolean.tsx

```diff
@@ -4,42 +4,39 @@ import { makeStyles } from "tss-react/mui";
 import { important } from "../../utils";
 
 export type CheckboxBooleanProps = {
-    onClick(value: boolean): void;
+    onClick(value: Boolean): void;
     id?: string;
     label?: string;
-    checked?: boolean;
+    checked?: Boolean;
```

Comment:
[same](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#the-primitives-string-number-and-boolean)